lib/api_resources/root.js

```diff
@@ -225,6 +225,10 @@ RootResource.prototype._renderRoot = function(env, next) {
         title: this.server._name,
         rel: [rels.server],
         href: env.helpers.url.path('/servers/' + encodeURI(this.server.id) )
+      },
+      {
+        rel: [rels.events],
+        href: env.helpers.url.path('/events').replace(/^http/, 'ws') 
```

Comment:
There are a few things to watch out for when building these URLs, such as... is there a forwarded protocol?  Is this an HTTP client connection or a SPDY client connection?  See here:  https://github.com/zettajs/zetta/blob/3a7eb1d7dcb69e91dc5c3c10bf0271a8faefd42f/lib/api_formats/siren/server.siren.js#L10-L15